docs/eventing/channels/channel-types-defaults.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Channel types and defaults"
+weight: 60
+type: "docs"
+showlandingtoc: "true"
+---
+
+Knative provides the InMemoryChannel channel implementation by default. This default implementation is useful for developers who do not want to configure a specific implementation type, such as Apache Kafka or NATSS channels.
```

Comment:
We mention in our Getting Started Docs that InMemoryChannels are great for prototyping around but probably shouldn't be used in production, do we want to repeat that sentiment here?

Reply:
Good idea. FWIW we can also later mention stuff like Kafka maybe more in the channel docs section, but I wanted to keep the scope for this PR mostly to cleaning up the content we already have, adding the diagram, etc.